### PR TITLE
Align updatedAt timestamp

### PR DESCRIPTION
**Pull request by ttbarnes**
- BSS/EWCS facilities now use `updatedAt` timestamp instead of `lastEdited`
- Various places: use `Date.now()`instead of legacy `now()` function (bonus: including TFM task timestamps)

This is required for better/simpler "All facilities" DB query.

### CODE REVIEW

File: dtfs-central-api/src/v1/swagger-definitions/portal/facility-bss.js [hunk 1]
```diff
@@ -16,8 +16,8 @@
  *       createdDate:
  *         type: string
  *         example: '1632392646415'
- *       lastEdited:
- *         type: string
+ *       updatedAt:
+ *         type: integer
  *         example: '1632392778918'
```
Comment (IgnatG) [inline dtfs-central-api/src/v1/swagger-definitions/portal/facility-bss.js:21]: minor thing:
if type is integer, then we can remove the quotes on this one, right?

Comment (ttbarnes) [reply dtfs-central-api/src/v1/swagger-definitions/portal/facility-bss.js]: Great spot, fixed

File: utils/mock-data-loader/bss/facilities/deal-9-facilities.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ module.exports = [
     'requestedCoverStartDate-year': twoMonths.year,
     bondBeneficiary: '',
     ukefExposure: '360,000.00',
-    lastEdited: nowTimestamp,
+    updatedAt: nowTimestamp,
```
Comment (IgnatG) [inline utils/mock-data-loader/bss/facilities/deal-9-facilities.js:71]: is this `nowTimestamp` being built using date-fns? or `Date.now()` and does it matter here which one do we use?
(this is for the other files listed above)

Comment (ttbarnes) [reply utils/mock-data-loader/bss/facilities/deal-9-facilities.js:71]: Yes it's using `Date.now()` from date-fns, just renamed it here

**Review by IgnatG — APPROVED** [review]
LGTM - nice job!

**Pull request merged**